packages/graph/hash_graph/lib/graph/src/ontology/types/entity_type/mod.rs

```diff
@@ -38,15 +33,15 @@ impl EntityTypeReference {
 }
 
 impl ValidateUri for EntityTypeReference {
-    fn validate_uri(&self, base_uri: &BaseUri) -> Result<(), ValidationError> {
-        ensure!(
-            base_uri == self.uri().base_uri(),
-            ValidationError::BaseUriMismatch {
+    fn validate_uri(&self, base_uri: &BaseUri) -> std::result::Result<(), ValidationError> {
+        if base_uri == self.uri().base_uri() {
+            Ok(())
+        } else {
+            Err(ValidationError::BaseUriMismatch {
                 base_uri: base_uri.clone(),
-                versioned_uri: self.uri().clone()
-            }
-        );
-        Ok(())
+                versioned_uri: self.uri().clone(),
+            })
+        }
```

Comment:
What is the reason for this change?

Reply:
`ValidateUri` is defined inside the Type System Package, which does not use `error-stack` currently.

Reply:
Gotcha, didn't noticed the `std::result::Result`, thanks for clarifying